addons/mail/static/src/core/common/settings_model.js

```diff
@@ -20,6 +20,7 @@ export class Settings extends Record {
         this._loadLocalSettings();
     }
     volumes = Record.many("Volume");
+    showOnlyVideo = false;
```

Comment:
Was already the case before your PR, so not to necessarily change it now, but `showOnlyVideo` is not persistent.

Users expect call settings menu to be persistent settings. If this was only for the call at hand, this should be in the call actions.

I think this was not made persistent because they were initially call actions (in "more" action) and have been moved to a dedicated panel. But we didn't made them persistent although the move of place would have implied making them persistent.